apps/site/tsconfig.json

```diff
@@ -3,7 +3,10 @@
   "references": [],
   "compilerOptions": {
     "outDir": "../../.moon/cache/types/apps/site",
-    "paths": {},
+    "baseUrl": ".",
```

Comment:
Is there any other reason for this besides that the imports are based on the root instead of relative to the file?

Reply:
personal preferences, i like using aliases

Reply:
I can live with the aliases, I don't really like that import paths are based on the root. AFAIK, path aliases work also without `baseUrl` in newer typescript version, but I would have find a docs for that. _So we can also keep and change later once I find the docs._

Reply:
I tried to remove `baseUrl` but it didn't work, does it require updates to `paths` as well?